compass-webapp/src/main/webapp/resources/js/editor/camera-controller.js

```diff
@@ -9,6 +9,75 @@ XML3D.tools.namespace("COMPASS");
 (function() {
 	"use strict";
 
+	COMPASS.NXExamineController = new XML3D.tools.Class(XML3D.tools.MouseExamineController, {
```

Comment:
For branding reasons, should we rename this to CADExamineController? Especially since it's not even the full functionality?